README.md

```diff
@@ -101,7 +101,33 @@ would in a Django shell.
 
 Unlike our other web applications this project is not using tox. To run tests locally, run this command instead:
 
+    ### PYTHON TESTS/LINTING
+    # Run Python tests
     docker-compose run --rm web pytest
```

Comment:
I would suggest updating `run --rm` to `exec` for all of these commands and specifying that if you do not already have a running container you should replace `exec` with `run --rm`. This would save some execution time for developers.